editor/src/components/inspector/sections/component-section/component-section.tsx

```diff
@@ -563,28 +567,37 @@ const RowForArrayControl = React.memo((props: RowForArrayControlProps) => {
                 )}
               </SquareButton>
             ) : null}
+            <ControlForProp
```

Comment:
this is the interesting change, RowForArrayControl renders ControlForProp which will render a Cartouche (if applicable) or return `null` if no cartouche was needed